src/Pages/Login/Login.js

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import Header from '../../Component/HeaderComponent/Header';
+import Footer from '../../Component/FooterComponent/Footer';
+
+import './Login.scss';
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      identity: '',
+      password: '',
+      background: '#d2f7d2',
+      disabled: true,
+      idChecked: false,
+      pwChecked: false,
+      isAvailedInfo: '',
+    };
+  }
+  //로그인 버튼활성화
+  // handleInputId = e => {
+  //   this.setState({ identity: e.target.value });
+  //   e.target.value.length > 4
+  //     ? this.setState({ idChecked: true }, () => this.btnChangeColor())
+  //     : this.setState({ idChecked: false }, () => this.btnChangeColor());
+  // };
+
+  // handleInputPw = e => {
+  //   this.setState({ password: e.target.value });
+  //   e.target.value.length > 7
+  //     ? this.setState({ password: e.target.value, pwChecked: true }, () =>
+  //         this.btnChangeColor()
+  //       )
+  //     : this.setState({ pwChecked: false }, () => this.btnChangeColor());
+  // };
+  handleInput = e => {
+    // input에 작성한 값
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  colorChange = () => {
+    if (this.state.identity.length > 4 && this.state.password.length > 4) {
+      this.setState({
+        background: '#6ca437',
+        disabled: false,
+      });
+    } else {
+      this.setState({
+        background: '#d2f7d2',
+        disabled: true,
+      });
+    }
+  };
+  // if(validation = 'true')
+
+  // btnChangeColor = () => {
+  //   this.state.idChecked && this.state.pwChecked
+  //     ? this.setState({ background: '#6ca437', disabled: false })
+  //     : this.setState({ background: '#d2f7d2', disabled: true });
+  // };
+  //버튼 이벤트 (백엔드 통신)
+  goToMain = () => {
+    fetch('http://10.58.7.179:8000/user/sign-in', {
+      method: 'POST',
+      body: JSON.stringify({
+        identity: this.state.identity,
+        password: this.state.password,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        console.log('결과: ', result);
+        //객체속 acess_token 값이 주어지면 로그인성공으로 페이지 이동
+        if (result.acess_token) {
+          alert('로그인 성공');
+          this.props.history.push('/main');
+        }
+      });
+  };
+  //회원가입 버튼 링크
+  goToSignUp = () => {
+    this.props.history.push('/signup');
+  };
+  render() {
+    let validation =
+      this.state.identity.length > 4 && this.state.password.length > 4;
+    return (
+      <div>
```

Comment:
- 위코드에서는 가장 바깥 태그에 컴포넌트와 동일한 이름으로 클래스 네임을 부여하시고, 이에 따라 네스팅 해주시는 것이 컨벤션입니다!
```suggestion
      <div className='login'>
```